src/generic3/templates/horizontal-pod-autoscaler.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ template "generic.fullname" . }}-hpa
+  labels:
+    app: {{ template "generic.name" . }}
+    chart: {{ template "generic.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: {{ .Values.resourceType }}
+    name: {{ template "generic.fullname" . }}
+  minReplicas: {{ .Values.replicaCount }}
```

Comment:
make this:
app:
  replica:
    enabled: false
    min:
    max:
    utilization:
      cpu: 70
      memory: 70
 or something similar